spec/models/teacher_spec.rb

```diff
@@ -1,5 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe Teacher, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it '#subject_name' do
+    subject = create(:subject, name: "Physics")
+    expect(subject.name).to eq('Physics')
```

Comment:
`expect(teacher.subject_name).to eq(subject.name)`

Reply:
fixed